src/cfnlint/rules/parameters/Used.py

```diff
@@ -26,7 +26,8 @@ class Used(CloudFormationLintRule):
     id = 'W2001'
     shortdesc = 'Check if Parameters are Used'
     description = 'Making sure the parameters defined are used'
-    tags = ['base', 'parameters']
+    sources = ['Best Practice']
```

Comment:
I was hoping sources was going to stay as URLs.  We could add these in Atom as links for error messages.

Reply:
Agree, see if I can fit it in